lib/capybara/server/middleware.rb

```diff
@@ -43,14 +47,14 @@ def call(env)
         if env['PATH_INFO'] == '/__identify__'
           [200, {}, [@app.object_id.to_s]]
         else
-          @counter.increment
+          @counter.increment(env)
```

Comment:
Can we just push the URI -- pushing the whole env seems wasteful and the server can modify the env which means it wouldn't match when decrementing